src/main/java/persistence/entity/EntityCash.java

```diff
@@ -0,0 +1,43 @@
+package persistence.entity;
+
+import common.ErrorCode;
+import jakarta.persistence.Id;
+
+import java.lang.reflect.Field;
+import java.util.Arrays;
+import java.util.Objects;
+
+public record EntityCash(Object key, Class<?> entityType) {
+    public EntityCash(Object entity) {
+        this(getKey(entity), entity.getClass());
+    }
```

Comment:
용도로 유추해봤을 때 Cache를 의도하셨을 것 같습니다.

```suggestion
public record EntityCache(Object key, Class<?> entityType) {
    public EntityCache(Object entity) {
        this(getKey(entity), entity.getClass());
    }
```

Reply:
부끄럽군요

Reply:
괜찮습니다ㅎㅎ
한글로는 둘다 캐시니까 충분히 헷갈릴 수 있는 부분이라고 생각합니다